server/services/file_search_operations.py

```diff
@@ -17,49 +44,65 @@ def extract_sections(file_path, search_terms, sections, specify_lines, use_total
         for line in lines:
             if term in line:
                 term_line_num.append(line_num)
+                print(f"Term: {term}, Line number: {line_num}, term_line_num: {term_line_num}")
                 terms_num += 1
             line_num += 1
 
+    search_term = search_terms[0]  #make generic later
```

Comment:
I added "#make generic later" as a sample comment for testing purposes. Please update it.